esp/scm/ws_dfu.ecm

```diff
@@ -319,6 +323,17 @@ DFUDefFileResponse
     [http_content("application/octet-stream")] binary defFile;
 };
 
+ESPrequest DFURecordTypeInfoRequest
+{
+    string Name;
+};
```

Comment:
Should we support:

  [min_ver("1.37")] bool IncludeJsonTypeInfo(true);
  [min_ver("1.37")] bool IncludeBinTypeInfo(true);